test/helper/request.go

```diff
@@ -82,6 +82,7 @@ func DoRequest(urlPath string, token string) (int, error) {
 
 	// #nosec
```

Comment:
Do we not need this line anymore?

Reply:
Yes, it's no longer needed. I'll remove it.

Reply:
Does this PR need to wait for #4934?